apps/lib/app_params.c

```diff
@@ -112,7 +112,7 @@ static void print_param_utf8(const char **s_ptr, size_t len)
     }
     BIO_puts(bio_out, "'");
     if (len > 0)
-        BIO_write(bio_out, s, len);
+        BIO_write(bio_out, s, (int)len);
```

Comment:
I assume that since we dont check any errors here we are fine with this truncating and doing nothing if the resulting truncation results in BIO_write doing nothing,,